docs/index.rst

```diff
@@ -141,6 +141,7 @@ and lots of other useful information.**
    experimental_features/modin_sql
    experimental_features/spreadsheets_api
    experimental_features/progress_bar
+   experimental_features/modin_in_the_cloud
```

Comment:
please add the link to this section in "Using Modin" section, too (it already has a very little section about Modin in the Cloud which needs to be polished as well).